pkg/controller/v1beta1/inferenceservice/components/transformer.go

```diff
@@ -100,12 +100,12 @@ func (p *Transformer) Reconcile(isvc *v1beta1.InferenceService) (ctrl.Result, er
 		}
 
 		// add predictor host and protocol to metadata
-		isvc.ObjectMeta.Annotations["predictor-host"] = predictorURL.Host
+		isvc.ObjectMeta.Annotations[constants.PredictorHostAnnotationKey] = predictorURL.Host
```

Comment:
@chinhuang007 It is also needed to be changed here 
https://github.com/kserve/kserve/blob/master/pkg/apis/serving/v1beta1/transformer_custom.go#L85